src/mixins/I13nMixin.js

```diff
@@ -108,8 +109,16 @@ var I13nMixin = {
         self._i13nNode.setDOMNode(ReactDOM.findDOMNode(self));
 
         // enable viewport checking if enabled
-        if (self._getReactI13n().isViewportEnabled()) {
-            self.subscribeViewportEvents();
+        if (reactI13n.isViewportEnabled()) {
+            var options = {};
+            if ( reactI13n.getScrollableContainerId &&
+                 reactI13n.getScrollableContainerId() ) {
+                if (reactI13n.getScrollableContainerDOMNode &&
```

Comment:
can we store `reactI13n.getScrollableContainerDOMNode()` in to a parameter ?  given that I13nMixin will be everywhere, should be better to reduce some unneeded executions